front/src/app/todo-list/todo-list.component.ts

```diff
@@ -54,6 +63,37 @@ export class TodoListComponent implements OnInit, OnDestroy {
     this.destroy$.complete();
   }
 
+  showNewTaskGroupFormField(): void {
+    this.isInputVisible = false;

```

Comment:
why in `showNewTaskGroupFormField` we make `input` not visible? I doesn't make sense to me